src/gui.cpp

```diff
@@ -934,6 +938,12 @@ void Gui_DrawCrosshair()
     qglDrawArrays(GL_LINES, 0, 4);
 }
 
+void Gui_DrawCrosshair()
```

Comment:
maybe, but stay here unimplemented empty function is not good too; this time will be used setting flag from config, so if crosshair does not needed, it could be disabled by config (+later by console command); when new crosshair be implemented, then Gui_DrawCrosshair() will be updated; do not needed for additional functions creation;